package.json

```diff
@@ -21,6 +21,7 @@
     "bootstrap": "3.3.6",
     "bootstrap-slider": "7.1.0",
     "closure-util": "1.15.1",
+    "moment-timezone": "^0.5.9",
```

Comment:
The problem with `^` is that you don't know which version you will get, because it installs the last major version. If you are trying to install the project, and in the mean time a new version has been released, you project might not work any more.

Feel free to update `angular-moment`.